src/Skia/Avalonia.Skia/SkiaSharpExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static SKFilterQuality ToSKFilterQuality(this BitmapInterpolationMode int
                     return SKFilterQuality.Medium;
                 case BitmapInterpolationMode.HighQuality:
                     return SKFilterQuality.High;
-                case BitmapInterpolationMode.Default:
+                case BitmapInterpolationMode.Unspecified:
```

Comment:
This removes the nearest neighbor interpolation mode that was previously mapped to Default.

Reply:
I guess we should remove Default and add a more descriptive value (with Default being an `[Obsolete]` alias

Reply:
Yes, but how to downgrade the interpolation mode to nearest neighbor? e. g.
```
<Border RenderOptions.BitmapInterpolationMode="HighQuality">
  <Image RenderOptions.BitmapInterpolationMode="Unspecified" />
</Border>
```
won't work.

Reply:
```
<Border RenderOptions.BitmapInterpolationMode="HighQuality">
  <Image RenderOptions.BitmapInterpolationMode="LowQuality" />
</Border>
```

Reply:
Ok, how do I set BitmapInterpolationMode to NearestNeighbor if there is no such value now?

Reply:
You can now disable interpolation via:
```
<Border RenderOptions.BitmapInterpolationMode="HighQuality">
  <Image RenderOptions.BitmapInterpolationMode="None" />
</Border>
```